modules/price/src/Element/Price.php

```diff
@@ -128,8 +128,8 @@ public static function processElement(array $element, FormStateInterface $form_s
       '#title' => $element['#title'],
       '#default_value' => $amount,
       '#required' => $element['#required'],
-      '#size' => $element['#size'],
-      '#maxlength' => $element['#maxlength'],
+      '#size' => $element['#amount_size'],
+      '#maxlength' => $element['#amount_maxlength'],
```

Comment:
Allows us to control size, maxlength in main element and not hardcode. But also not trigger the `Unicode::strlen()`